cells/_automation/devshells.nix

```diff
@@ -0,0 +1,33 @@
+{
+  cell,
+  inputs,
+}: let
+  inherit (inputs.nixpkgs) lib;
+  inherit (inputs.std) std presets;
+  inherit (inputs.std.lib) dev;
+  inherit (inputs.cells) prelude;
+  inherit (prelude.library) project;
+
+  mkDevShell = project:
+    dev.mkShell {
+      name = "Cardano Prelude - ${project.args.compiler-nix-name}";
+      imports = [
+        std.devshellProfiles.default
+        (prelude.devshellProfiles.mkDev project)
+      ];
+      nixago = lib.attrValues cell.nixago;
+      commands = [
+        {
+          package =
+            inputs.tullia.tullia.apps.tullia
+            // {meta.description = "CLI for Cicero tasks and actions";};
```

Comment:
Can we fix this upstream?

Reply:
right, forgot to do it. https://github.com/input-output-hk/tullia/pull/7

Reply:
Merged :heavy_check_mark: